### ISSUE TITLE
Units for consequence tables

### ISSUE THREAD

**Issue by jeffreyhanson** [NCC-CNC/whattemplatemaker#11]
Veronique suggest it would be useful to allow users to specify the units for each feature in the consequence table. What do you think @ricschuster? This has the advatange that the excel spreadsheet could display units - but the disadvantage that users won't be able to change the units later on if needed?

**Comment by ricschuster** [comment]
I don't really have a strong opinion either way, although I'd prefer to allow users to change units later on if needed.
How about you @jeffreyhanson?

**Comment by jeffreyhanson** [comment]
Hmm, we could add a "Units" column to the Feature data sheet (e.g., after the relative weights columns, see below) that is available for editing? The advatantage with this is that it can be changed whenever and the app could be updated to display these units. The disadvatange is that the units won't appear in the popup/comment text for cells in the spreadsheet.

![image](https://user-images.githubusercontent.com/3610005/153771784-a11e33b5-0561-4c34-b892-2e8417c6f58a.png)

**Comment by ricschuster** [comment]
I like this idea. I don't see this as a high priority though (nobody has mentioned this so far). Unless you think this should be higher priority, we could add 'sometime' to this and park it for now. What do you think?

**Comment by jeffreyhanson** [comment]
Yeah I agree - I think there's other stuff that should be a higher priority than this.

**Comment by jeffreyhanson** [comment]
In the meantime, user's can mention the units for the features in the description information.

**Comment by ricschuster** [comment]
Sounds great, thanks!
